frontend/src/features/contacts/contact/create/Organization/CreateOrganizationForm.tsx

```diff
@@ -105,7 +104,6 @@ export const CreateOrganizationForm: React.FunctionComponent = () => {
       <Formik
         component={CreateOrganizationComponent}
         initialValues={defaultCreateOrganization}
-        enableReinitialize
```

Comment:
question - why you took this away? most forms have it but wondering why was needed to remove here? did you find something odd?

Reply:
Oops I can add that back in this was while I was debugging! Thanks for catching